packages/metal-state/src/Config.js

```diff
@@ -110,12 +110,82 @@ function mergeConfig(context, config) {
 	return obj;
 }
 
-// Add all validators to `Config`.
-const fnNames = Object.keys(validators);
-fnNames.forEach(
-	name => Config[name] = function() {
+/**
+ * Adds primitive type validators to the config object.
+ * @param {string} name The name of the validator.
+ */
+function setPrimitiveValidators(name) {
+	Config[name] = function() {
 		return this.validator(validators[name]);
-	}
-);
+	};
+}
+
+setPrimitiveValidators('any');
```

Comment:
Yes, it would be the same.

It would just make this file much easier to read, since all methods on `Config` would be explicitly declared in the same way.